galaxy_ng/app/access_control/statements/pulp.py

```diff
@@ -195,6 +195,20 @@
     "pulp_ansible/v3/collection-versions/docs": _collection_statements,
     "pulp_ansible/v3/collections/imports": _collection_statements,
     "pulp_ansible/v3/repo-metadata": _collection_statements,
+    "content/ansible/namespaces": {
+        "statements": [
+            {
+                "action": ["list", "retrieve"],
+                "principal": "authenticated",
+                "effect": "allow",
+            },
+            {
+                "action": "avatar",
+                "principal": "*",
+                "effect": "allow",
```

Comment:
is there a reason the `principal` is different on these 2 read namespace actions?

Reply:
yes. I wanted the avatar to be accessible to unauthenticated clients.